examples/angular/src/app/pages/pages.component.ts

```diff
@@ -41,17 +41,15 @@ export class DotCMSPagesComponent implements OnInit, OnDestroy {
   protected readonly components = signal<any>(DYNAMIC_COMPONENTS);
   private readonly client = inject(DOTCMS_CLIENT_TOKEN);
 
+  protected readonly editorCofig = { params: { depth: '2' } };
+
   ngOnInit() {
     // Get the context data from the route
     this.route.data
       .pipe(takeUntilDestroyed(this.destroyRef))
       .subscribe((data) => {
         this.context.set(data['context']);
       });
-
-    this.client.editor.on('changes', (pageAsset) => {
```

Comment:
Is this not longer necessary?
I see this was moved to useDotcmsEditor, but the changes can be reflected at the "pages" (client application) level?

Reply:
That's right.

Now, we are listening to those changes from the `useDotcmsEditor`. Every time a change is made in the UVE, it gets reflected at the user level.

Reply:
Same with Angular!

Reply:
@KevinDavilaDotCMS 

https://github.com/user-attachments/assets/f2e9be3f-9f1c-472f-86d6-72bce1c5ac1d